src/App.css

```diff
@@ -1,3 +1,53 @@
 body {
+  background-color: #fafafa;
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  padding-left: 20px;
+  color: #0893ff;
+  font-weight: 500;
+  background-color: #fff;
+}
+
+.nav-links {
+  display: flex;
+  align-items: center;
+  color: black;
+  font-weight: 300;
+  font-size: 120%;
+}
+
+.nav-links > * {
+  margin-left: 10px;
+  color: black;
+  text-decoration: none;
+}
+
+#add-book-form {
+  padding: 10px 50px;
+}
+
+#add-book-form > input {
+  margin-right: 20px;
+  padding: 5px;
+}
+
+#add-book-form > button {
+  margin-right: 20px;
+  padding: 5px 10px;
+  background-color: #0290ff;
+  color: #fafafa;
+  border: none;
+}
```

Comment:
- [ ] You want to consider following the form's design too. ⚒️

**Yours**
![image](https://user-images.githubusercontent.com/79876959/145666677-36684290-7699-4baf-9fc8-c152afd30bf8.png)

**Template**
![image](https://user-images.githubusercontent.com/79876959/145666685-cf619fc5-ec97-4f39-80ab-8001bca5beeb.png)